### PR TITLE
suggested update to machineset definition for labels

### PR DESCRIPTION
**Pull request by kalexand-rh**
Per Jake Lucky in Slack.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@sunzhaohua2, will you PTAL?

**Comment by kalexand-rh** [comment]
@jhou1, will you PTAL?

File: modules/machineset-yaml-aws.adoc [hunk 1]
```diff
@@ -39,7 +41,7 @@ spec:
     spec:
       metadata:
         labels:
-          node-role.kubernetes.io/<role>: "" <3>
+          node-role.kubernetes.io/<role>: "true" <2>
```
Comment (jhou1) [inline modules/machineset-yaml-aws.adoc]: This is the node label to add. So `<2>` should be `<3>`

File: modules/machineset-yaml-aws.adoc [hunk 2]
```diff
@@ -28,6 +28,8 @@ spec:
   selector:
     matchLabels:
       machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+      machine.openshift.io/cluster-api-machine-role: <role> <3>
+      machine.openshift.io/cluster-api-machine-type: <role> <3>
```
Comment (jhou1) [inline modules/machineset-yaml-aws.adoc]: `<3>` should be `<2>` for above 2 lines

File: modules/machineset-yaml-aws.adoc [hunk 3]
```diff
@@ -84,6 +86,6 @@ command:
 ----
 $ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 ----
-<2> Specify the infrastructure ID, node label, and zone.
+<2> Specify the infrastructure ID, node label, and zone. Use a descriptive name for the node label, such as `infra` for infrastructure machines.
```
Comment (jhou1) [inline modules/machineset-yaml-aws.adoc]: Use a descriptive name for the **machine** label,

File: modules/machineset-yaml-azure.adoc [hunk 4]
```diff
@@ -88,7 +88,7 @@ command:
 ----
 $ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 ----
-<2> Specify the node label to add.
+<2> Specify the node label to add, such as `infra` for infrastructure machines.
```
Comment (jhou1) [inline modules/machineset-yaml-azure.adoc]: such as `infra` for infrastructure **nodes**.

**Comment by jhou1** [comment]
Hi @kalexand-rh, in a machineSet, the labels nested like the following will be applied to the nodes.

```
spec:
  template:
...
    spec:
      metadata:
        labels:
          node-role.kubernetes.io/<role>: "true"
```

File: modules/machineset-yaml-aws.adoc (hunk 2 above)
Comment (jhou1) [reply modules/machineset-yaml-aws.adoc]: Actually no need to update here, if we want the machines to be labeled, we should update 

```
spec:
  template:
    metadata:
      labels:
```

**Pull request closed without merging**